city_scrapers/spiders/chi_ssa_60.py

```diff
@@ -0,0 +1,129 @@
+import html
+import json
+from datetime import datetime
+
+from city_scrapers_core.constants import ADVISORY_COMMITTEE, COMMISSION, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.relativedelta import relativedelta
+from scrapy import Request
+
+
+class ChiSsa60Spider(CityScrapersSpider):
+    name = "chi_ssa_60"
+    agency = "Chicago Special Service Area #60 Albany Park"
+    timezone = "America/Chicago"
+    months_backward = 10
+    months_forward = 1
+
+    start_urls = None  # Generated by `start_requests`
+    custom_settings = {'ROBOTSTXT_OBEY': False}
+
+    def start_requests(self):
+        requests = []
+        for i in range(-self.months_backward, self.months_forward + 1):
+            dt = datetime.strftime(datetime.now().date() + relativedelta(months=i), "%Y-%m")
+            requests.append(
+                Request(
+                    "https://northrivercommission.org/events/{}/".format(dt), callback=self.parse
+                )
+            )
+        return requests
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        event_list = list()
+        for item in response.xpath("//script[@type='application/ld+json']/text()"):
```

Comment:
You could do something like `extract_first` or just using the first index here instead of iterating through just to make it more clear that it should only find one thing

Reply:
That's a good idea.